yr/scripts/tutorialPage.js

```diff
@@ -243,6 +243,11 @@ var setupTutorial = function() {
     updateTutorialPage(tutorialIndex);
     return false;
   });
+
+  // Add show/hide hint functionality
+  $(".hintContainer").off("click").on("click", function(e) {
+    $(".hint").toggleClass("hideHint");
```

Comment:
This selects all elements with the hint class. In this particular example, there is only one, so it works fine. But if we ever have more than one hint this will toggle the visibility for all hints. You can scope the `.hint` selector by passing `this` as a second argument to `$`. Note that if you make the change I suggest above, you will need to do `this.parentElement` since `this`.